tools/astarte_import/test/test_helper.exs

```diff
@@ -1 +1,9 @@
 ExUnit.start()
+
+children = [
+  {Astarte.DataAccess, xandra_options: Astarte.DataAccess.Config.xandra_options!()}
+]
```

Comment:
It would be better to use a different Xandra pool just for tests, so that we're not using DataAccess' resource.
 ```suggestion
    xandra_options =
      Astarte.DataAccess.Config.xandra_options!()
      |> Keyword.put(:name, :test_xandra)

    children = [
      {Xandra.Cluster, xandra_options}
    ]
```

This would mean that all calls to `Xandra.Cluster` functions should use the new pool, see comment above for an example